packages/core/src/Stores/client-store.js

```diff
@@ -242,7 +242,8 @@ export default class ClientStore extends BaseStore {
         if (['set_currency', 'manage'].includes(this.root_store.ui.real_account_signup_target)) {
             return this.current_landing_company.legal_allowed_currencies;
         }
-        const target = this.root_store.ui.real_account_signup_target === 'maltainvest' ? 'financial' : 'gaming';
+        const target =
+            this.root_store.ui.real_account_signup_target === 'maltainvest' || this.is_au ? 'financial' : 'gaming';
```

Comment:
We should not be hardcoding logic based on `residence`. With the current logic, it breaks for the countries which don't have `gaming`.
Later if we block `gaming` for some other countries from BO, it will break again.